apps/src/dropletUtils.js

```diff
@@ -158,52 +158,44 @@ standardConfig.categories = {
  * @returns {Array}
```

Comment:
It would be helpful to add a type param for codeFunctions, and (though it may not be 100% standard) dropletConfig.blocks